### DISCUSSION TITLE
How to deploy flutter web on globe?

### DISCUSSION THREAD

**Discussion by ClarissaGazalaEvanthe** [invertase/globe#17] [Q&A] [answered]
I am trying to create a flutter hello world web,
but an error during build when I press it doesn't bring up anything
![Screenshot from 2023-12-15 23-44-54](https://github.com/invertase/globe/assets/84509715/d5312f67-80c4-41b5-add4-ecb19f9c90cb)

**Comment by ClarissaGazalaEvanthe** [comment]
![Screenshot from 2023-12-16 01-37-34](https://github.com/invertase/globe/assets/84509715/3352a2ec-5fe6-470d-bb17-b3ba2e338bf7)
I tried again to build successfully but the website doesn't appear. How about this?

**Comment by lesnitsky** [comment, marked as answer]
@gwcangtip we had an issue with flutter web deployments, which is now fixed, so flutter deployments should work now.

**Reply by ClarissaGazalaEvanthe** [reply]
thanks
